jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/editor/LogStreamTable.kt

```diff
@@ -59,6 +60,7 @@ class LogStreamTable(
         logsTable.columnModel.getColumn(0).maxWidth = 150
         logsTable.autoResizeMode = JTable.AUTO_RESIZE_LAST_COLUMN
 
+        TableSpeedSearch(logsTable)
```

Comment:
does this reference not get garbage collected?

Reply:
`TableSpeedSearch` installs itself onto the component passed in. It's pretty gross.